app/auth/views.py

```diff
@@ -62,16 +68,19 @@ def post():
                         prev = letter
                         count = 0
 
-                user.save()
-                # msg = Message(
-                #     "welcome to Bright Events",
-                #     sender="tonny.nesh@gmail.com",
-                #     recipients=["tonnie.nesh@gmail.com"] #user.email
-                # )
+                msg = Message(
+                    "welcome to Bright Events",
+                    sender="bryt.event@gmail.com",
+                    recipients=["tonnie.nesh@gmail.com"] #user.email
+                )
 
-                # msg.body = 'Thank you for creating an account with. Please login to access the features of the application'
+                
+                html = render_template("inline_welcome.html", name=name)
+                msg.html = html
 
-                # mail.send(msg)
+                MAIL.send(msg)
```

Comment:
it would be nice to first save the user before sending out the email. what if the user is not saved for one reason or another? an error when saving maybe.

Reply:
What if there is an error in sending the email and the user gets to be saved, we could not fix the error and register the user, thats why first I send the email then save the user if the email is sent successfully

Reply:
your argument is valid. However, my thought on this is that, in the case of database failure, and it happens that the user was not saved but the email was sent, will the user be confirming a non-existing account?

Reply:
All the error handlers for saving a user to the database are before the email being sent, incase of any error they will be raised before the email code is read

Reply:
cool. then it's fine that way.